### PR TITLE
fix(web) small UI improvements

### PR DESCRIPTION
**Pull request by faupau03**
- add slight background asset viewer navbar so that buttons are more visible
- scroll only the page content and not the sidebar
- fix map overlay sidebar, when sidebar opens after being collapsed

fixed
normal
fixed-button
normal-button
map overlay fixed
map overlay normal

### CODE REVIEW

**Comment by alextran1502** [comment]
Can you please add screenshots to where applicable. Thank you

File: web/src/lib/components/asset-viewer/asset-viewer-nav-bar.svelte [hunk 1]
```diff
@@ -88,31 +92,44 @@
...
+				<CircleIconButton
+					isOpacity={true}
+					logo={DotsVertical}
+					on:click={showOptionsMenu}
+					title="More"
+				>
 					{#if isShowAssetOptions}
 						<ContextMenu {...contextMenuPosition}>
-							<div class="flex flex-col rounded-lg ">
+							<div class="flex flex-col rounded-lg text-black bg-immich-bg" title="">
```
Comment (michelheusschen) [inline web/src/lib/components/asset-viewer/asset-viewer-nav-bar.svelte:132]: Empty title attribute can be removed

File: web/src/lib/components/layouts/user-page-layout.svelte [hunk 2]
```diff
@@ -27,21 +29,23 @@
...
-					<hr class="dark:border-immich-dark-gray" />
-				</div>
-			{/if}
-
-			<slot />
-		</section>
-	</slot>
+	<ConditionalWrapper condition={!isTimeline} class="overflow-y-auto">
+		<slot name="content">
+			<section class="my-8 mx-4 bg-immich-bg dark:bg-immich-dark-bg">
```
Comment (michelheusschen) [inline web/src/lib/components/layouts/user-page-layout.svelte:34]: Why is this conditional wrapper even needed? The photos page overrides the content slot, so adding `overflow-y-auto` to the section element should work

File: web/src/lib/components/asset-viewer/asset-viewer-nav-bar.svelte (hunk 1 above)
Comment (faupau03) [reply web/src/lib/components/asset-viewer/asset-viewer-nav-bar.svelte]: Done

File: web/src/lib/components/layouts/user-page-layout.svelte (hunk 2 above)
Comment (faupau03) [reply web/src/lib/components/layouts/user-page-layout.svelte]: You are right ;)
Done

**Comment by michelheusschen** [comment]
The z-indexes are still not quite right, because the modal backdrop no longer covers the navigation bar and sidebar. If only the map is overlapping the sidebar, you can also create a new stacking context by adding `relative z-0` to the [map wrapper](https://github.com/immich-app/immich/blob/d91247dc35aeef5f7d461231d8c54e6e64e4396a/web/src/routes/(user)/map/%2Bpage.svelte#L48). Then the changes you made to z-index are no longer needed.

**Review by alextran1502 — APPROVED** [review]

**Pull request merged**